lib/Lolifiers/Blackout.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Monolol\Lolifiers;
+
+
+use Monolol\Lolifier;
+
+class Blackout implements Lolifier
+{
+    const MESSAGE = ' ... hum ... i forgot the rest of this sentence';
+
+    public function isHandling(array $record)
+    {
+        return true;
+    }
+
+    public function lolify(array $record)
+    {
+        $record['message'] = $this->forgotHalfSentence($record['message']);
+
+        return $record;
+    }
+
+    private function forgotHalfSentence($message)
+    {
+        $nbWords = str_word_count($message);
+
+        if($nbWords > 1)
+        {
+            $posHalfSentence = floor(strlen($message) / 2);
+            $message = substr($message, 0, strpos($message, ' ', $posHalfSentence)) . self::MESSAGE;
```

Comment:
it doesn't. It just really cut on the word before the deadline (the half of the sentence), with that being *any* blank caracter (e.g a space, a tab, a new line, ...). If my log line is the following :

```
log
line
pretty
long
```

this lolifier will output the *whole* sentence, rather than the middle (```log\nline```)